docs/reason.md

```diff
@@ -35,9 +35,10 @@ robot reason --reasoner ELK \
   --output results/reasoned.owl
 ```
 
-Finally, `reason` includes two more options to help clean the reasoned output:
+Finally, `reason` includes three more options to help clean the reasoned output:
 * `--exclude-duplicate-axioms`: if set to true, the axioms will not be added to the output if they exist in an import (default `false`).
 * `--remove-redundant-subclass-axioms`: if set to false, redundant axioms (those that have been asserted and were also inferred) are included in the output (default `true`).
+* `--exclude-tautologies`: if set to 'structural', exclude axioms matching a hard-coded set of tautological patterns (e.g., `X SubClassOf owl:Thing`, `owl:Nothing SubClassOf X`, `X SubClassOf X`); if 'all', use the HermiT reasoner to exclude any inferred axioms that would be entailed by an empty ontology; if 'false' (default), include any tautological inferences that may be asserted by an axiom generator.
```

Comment:
I think this could use a bit more description in it's own section, rather than just a single bullet point. This will probably only be used by more advanced users, but somebody may be reading the docs and not understand what a tautological pattern is and what a "hard-coded" set of these patterns is.

Reply:
Done.